mne/proj.py

```diff
@@ -384,9 +385,15 @@ def sensitivity_map(fwd, projs=None, ch_type='grad', mode='fixed', exclude=[],
     if mode in ['fixed', 'free']:
         sensitivity_map /= np.max(sensitivity_map)
 
-    vertices = [fwd['src'][0]['vertno'], fwd['src'][1]['vertno']]
     subject = _subject_from_forward(fwd)
-    stc = SourceEstimate(sensitivity_map[:, np.newaxis],
-                         vertices=vertices, tmin=0, tstep=1,
-                         subject=subject)
+    if len(fwd['src']) == 1:  # volume source space
```

Comment:
This is not a robust check because it could be discrete, or mixed (2+ source spaces) not just 1=vol 2=surf. You want to query `src['type']`